modules/javafx.graphics/src/test/jslc/com/sun/scenario/effect/compiler/parser/PrimaryExprTest.java

```diff
@@ -58,14 +58,14 @@ public void variable() throws Exception {
     public void intLiteral() throws Exception {
         Expr tree = parseTreeFor("123");
         assertTrue(tree instanceof LiteralExpr);
-        assertEquals(((LiteralExpr)tree).getValue(), new Integer(123));
+        assertEquals(((LiteralExpr)tree).getValue(), Integer.valueOf(123));
     }
 
     @Test
     public void floatLiteral() throws Exception {
         Expr tree = parseTreeFor("1.234");
         assertTrue(tree instanceof LiteralExpr);
-        assertEquals(((LiteralExpr)tree).getValue(), new Float(1.234));
+        assertEquals(((LiteralExpr)tree).getValue(), Float.valueOf(1.234));
```

Comment:
We must not be  building or running these tests. This won't compile without casting to `float` or using a float constant, `1.234f`.